packages/admin-panel/src/autocomplete/Autocomplete.js

```diff
@@ -1,198 +1,133 @@
-/**
- * Tupaia MediTrak
- * Copyright (c) 2018 Beyond Essential Systems Pty Ltd
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2023 Beyond Essential Systems Pty Ltd
  */
 
 import React from 'react';
 import PropTypes from 'prop-types';
-import throttle from 'lodash.throttle';
-import { connect } from 'react-redux';
-import MuiChip from '@material-ui/core/Chip';
 import { createFilterOptions } from '@material-ui/lab/Autocomplete';
 import styled from 'styled-components';
-import { Autocomplete as AutocompleteBase } from '@tupaia/ui-components';
-import { getAutocompleteState } from './selectors';
-import { changeSelection, changeSearchTerm, clearState } from './actions';
+import MuiChip from '@material-ui/core/Chip';
+import { Autocomplete as UIAutocomplete } from '@tupaia/ui-components';
```

Comment:
I wonder if by convention, the name should be UiAutocomplete. Having the first word in caps doesn't seem to fall in line with the way we name react components

Reply:
We will do that when it comes to abbreviation, e.g. PDFExportPage